src/flagsmith/mod.rs

```diff
@@ -358,6 +361,23 @@ fn get_environment_from_api(
     return Ok(environment);
 }
 
+fn update_environment(
+    client: &reqwest::blocking::Client,
+    datastore: &Arc<Mutex<DataStore>>,
+    environment_url: &String,
+) -> Result<(), error::Error> {
+    let mut data = datastore.lock().unwrap();
+    let environment = Some(get_environment_from_api(
+        &client,
+        environment_url.clone(),
+    )?);
+    for identity in &environment.as_ref().unwrap().identity_overrides {
+        data.identities_with_overrides_by_identifier.insert(identity.identifier.clone(), identity.clone());
```

Comment:
I'll happily accept pointers on how to stop copying around the identity structs. This shouldn't be a problem for most environments, but we need to be aware there's room for optimisation here.